charts/govuk-rails-app/values.yaml

```diff
@@ -14,9 +14,8 @@ ingress:
     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
     alb.ingress.kubernetes.io/ssl-redirect: "443"
     alb.ingress.kubernetes.io/healthcheck-path: /healthcheck/ready
-  hosts:
-  - name:
-    path: /
+  rules:
+  - path: /
```

Comment:
I think you're missing the `http` and `paths` here. See `kubectl explain ing.spec.rules` - the next level down has to be either a `host` (which we don't want any more of course) or an `http`. Then `kubectl explain ing.spec.rules.http` tells us that `http` has to contains a `paths`, which is a list of `path`. Here's a [full example](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/ingress/cert_discovery/#discover-via-ingress-tls).

Reply:
Oops sorry, I initially misread this as the spec rather than the Helm value being passed in, then when I noticed it was the Helm value I didn't go back and check it against the template. D'oh. You're absolutely right - it's all good.